### PR TITLE
Rework MemSQL tests to use BaseConnectorTest

### PR DESCRIPTION
**Pull request by alexjo2144**

### CODE REVIEW

**Review by findepi — APPROVED** [review]

**Comment by findepi** [comment]
@alexjo2144 see build red

**Comment by alexjo2144** [comment]
Fixed, thanks. The issue was two tests with the same name once I combined the two.

File: plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlConnectorTest.java [hunk 1]
```diff
@@ -78,7 +175,7 @@ public void testViews()
     }
 
     @Test
-    public void testInsert()
+    public void testSimpleInsert()
```
Comment (findepi) [inline plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlConnectorTest.java:178]: Is this a duplicate of `testInsert` and can be removed?

Comment (alexjo2144) [reply plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlConnectorTest.java]: Looks like the insert tests is a dup, I'll remove it. There's also a `testViews` which makes me think that views are supported, so I tried enabling those tests too.

**Comment by findepi** [comment]
@kokosing @s2lomon please see `TestOAuth2WebUiAuthenticationFilter.setup` failure in https://github.com/trinodb/trino/runs/2024726693

**Pull request merged**